ee/dtr/admin/disaster-recovery/create-a-backup.md

```diff
@@ -76,36 +89,48 @@ recommended for that system.
 ### Back up DTR metadata
 
 To create a DTR backup, load your UCP client bundle, and run the following
-command, replacing the placeholders for the real values:
+command. For your convenience, this command automatically populates your DTR version and replica ID:
 
 ```none
+DTR_VERSION=$(docker container inspect $(docker container ps -f name=dtr-registry -q) | \
+  grep -m1 -Po '(?<=DTR_VERSION=)\d.\d.\d'); \
+REPLICA_ID=$(docker ps --filter name=dtr-rethinkdb --format "{{ .Names }}" | head -1 | \
```

Comment:
@omegamormegil This is what @caervs recommends for setting your DTR replica ID: 
```
REPLICA_ID=$(docker inspect -f '{{.Name}}' $(docker ps -q -f name=dtr-rethink) | cut -f 3 -d '-')
```

I'll have him review the other example commands to this PR. Thank you so much for your contribution yet again.